audiobook/src/test/java/de/ph1b/audiobook/features/chapterReader/OggReadingTest.kt

```diff
@@ -0,0 +1,253 @@
+package de.ph1b.audiobook.features.chapterReader
+
+import org.assertj.core.api.Assertions.*
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import java.io.File
+import java.util.*
+
+private fun unsignedByteArrayOf(vararg values: Int) = values.map { it.toByte() }.toByteArray()
+
+private fun <T> Iterable<T>.shuffle(random: Random? = null): List<T> {
```

Comment:
My idea was that someone might want to pass always the same seeded instance of `Random` to have deterministic values, but yeah, in this place, it doesn't really matter. Done.